HotelBooking/Views/customer_view.py

```diff
@@ -25,7 +26,8 @@
 class CustomerView(View):
     user_id: str
     view_options: List[Tuple[str, View]] = [
-        ("Book/Cancel reservation", ReservedRoomsView),
+        ("Book reservation", BookReservationView),
+        ("Cancel reservation", CancelReservationView),
```

Comment:
Is admin still able to cancel reservation?

Reply:
checking now,  but the only change I made to admin.py was removing an unused import so I think so

Reply:
![image](https://user-images.githubusercontent.com/50186952/199795813-55b81605-a871-4339-b24b-1d1771374cf1.png)
looks like it still works